FlowCrypt/src/main/java/com/flowcrypt/email/security/pgp/PgpMsg.kt

```diff
@@ -1304,6 +1305,92 @@ object PgpMsg {
     )
   }
 
+  private fun checkAndReturnQuotesFormatIfFound(content: String): String? {
+    return buildQuotes(originalContent = content, unwrapContent = false)?.outerHtml()
+  }
+
+  private fun buildQuotes(originalContent: String, unwrapContent: Boolean = true): Element? {
+    val content = if (unwrapContent) {
+      //remove > at the beginning of all lines to define next quotes level
+      val patternQuotesSign = "^>([^\\S\\r\\n])?".toRegex(RegexOption.MULTILINE)
+      originalContent.replace(patternQuotesSign, "")
+    } else {
+      originalContent
+    }
+
+    val newLineStringPattern = "\\r\\n|\\r|\\n"
+    val beforeQuotesHeaderStringPattern = "^.*:($newLineStringPattern){1,2}"
```

Comment:
What if a user enters something like this? The code would detect “Below you can find information about something:” as a pre-quotes header, I think.  
I think we shouldn’t only check if there’s a colon (“:”) before a new line, but we should also look for something like “On ... Wrote:” before a new line.  
What do you think?

```
Below you can find information about something:
Some info

On 2025-02-20 at 13:13, Den<denbond7@flowcrypt.test> wrote:
...
```

Reply:
Ohhh. yep
As I remember we used  “On ... Wrote:” logic in browser extension i think and it would be problem for different languages.

I think we should live with `:` as you suggested.

Reply:
>As I remember we used “On ... Wrote:” logic in browser extension i think and it would be problem for different languages.
>I think we should live with : as you suggested.

I've started to work on encrypted replies. I will compare all clients and will add more details in a separate issue.